style.css

```diff
@@ -0,0 +1,550 @@
+  display: grid;
+  grid-template-columns: 25px 1fr;
+  place-content: center;
+}
+
+.search-rounded p {
+  display: inline;
+  margin-bottom: 5px;
+}
+
+.main-feed-post-area {
+  display: grid;
+  grid-template-columns: 55px 1fr;
+  gap: 5px;
+}
+
+.post-input-area {
+  display: grid;
+  grid-template-rows: 50px 50px;
+  gap: 15px;
+}
+
+.post-input-area p {
+  font-size: 21px;
+  color: #867d7d;
+}
+
+.post-type-icons {
+  display: grid;
+  grid-template-columns: 1fr 90px;
+  gap: 45px;
+}
+
+.post-type-icons button {
+  padding: 11px 21px;
+  border: none;
+  color: white;
+  font-weight: bold;
+  border-radius: 50px;
+  background-color: #0d95d4;
+  opacity: 0.5;
+}
+
+.post-icons {
+  display: grid;
+  grid-template-columns: repeat(6, 20px);
+  gap: 15px;
+}
+
+.post-icons img {
+  height: 19px;
+  filter: invert(43%) sepia(34%) saturate(3744%) hue-rotate(171deg)
+    brightness(95%) contrast(90%);
+}
+
+.main-feed-show-more {
+  display: grid;
+  place-content: center;
+  margin: 15px;
+}
+
+.main-feed-show-more a {
+  color: #0d95d4;
+  text-decoration: none;
+  font-weight: 400;
+}
+
+.feeds-thread {
+  display: grid;
+  grid-template-columns: 55px 1fr;
+  margin: 30px 0 55px 0;
+}
+
+.post-section {
+  display: grid;
+  grid-template-rows: 110px 55px 1fr;
+}
+
+.content-area .div1 {
+  display: grid;
+  grid-template-columns: 1fr 20px;
+}
+
+.content-area div p {
+  font-size: 14px;
+}
+
+.content-area div p :nth-child(2) {
+  color: rgb(138, 136, 136);
+}
+
+.content-area img {
+  height: 15px;
+  filter: invert(63%) sepia(0%) saturate(31%) hue-rotate(198deg) brightness(81%)
+    contrast(83%);
+}
+
+.content-area .div2 span {
+  color: #0d95d4;
+}
+
+.content-area .div3 > img {
+  margin-top: 15px;
+  width: 70%;
+  height: auto;
+  object-fit: cover;
+  filter: none;
+  border-radius: 11px;
+}
+
+.content-area .div4 {
+  margin-top: 11px;
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(20px, 1fr));
+  font-weight: 400;
+}
+
+.content-area .div4 img {
+  height: 16px;
+  margin-right: 5px;
+}
+
+.content-area .div4 span {
+  font-size: 15px;
+}
+
+.post-image-avatar div {
+  height: 100%;
+  width: 1px;
+  border: #a8b9c5 solid 1px;
+  display: grid;
+  align-self: center;
+  margin-left: 17px;
+}
+
+@media screen and (max-width: 1260px) {
+  .header-space {
+    grid-template-columns: 90px;
+  }
+
+  .menu-list-content b {
+    display: none;
+  }
+
+  .tweet-button button {
+    font-size: 11px;
+    border-radius: 50px;
+  }
+
+  .user-name,
+  .handle,
+  .more-icon {
+    display: none;
+  }
+
+  .header-bottom-side {
+    display: grid;
+    justify-content: center;
+  }
+
+  .user-avatar {
+    margin-left: 15px;
+  }
+}
+
+@media screen and (max-width: 1100px) {
+  .main-space {
+    display: grid;
+    grid-template-columns: 1fr;
+    gap: 0px;
+  }
+
+  .main-feed-divider {
+    display: none;
+  }
+}
+
+@media screen and (max-width: 700px) {
+  .top-main-menu {
+    width: 450px;
+  }
+}
```

Comment:
Clear divisions of the CSS styling! It is very lengthy, there might be some chances to condense the code by grouping together some of the properties you have repeated? (eg - gaps, margins etc...)